pipenv/utils/shell.py

```diff
@@ -242,7 +242,7 @@ def is_virtual_environment(path):
     for bindir_name in ("bin", "Scripts"):
         for python in path.joinpath(bindir_name).glob("python*"):
             try:
-                exeness = is_file() and os.access(str(python), os.X_OK)
+                exeness = python.is_file() and os.access(str(python), os.X_OK)
```

Comment:
This was a bug from my utils splitting (thanks pycharm import refactor) and I have ported this fix back to that branch which is still out for PR.